frontend/src/components/dashboard/ArtworkCloseUpCard.js

```diff
@@ -54,63 +61,73 @@ export default function ArtworkCloseUpCard() {
   useEffect(() => {
     const params = new URLSearchParams();
     params.append('text', description);
-    params.append('id', 'picture-id');
+    params.append('id', props.match.params.id);
     fetch('/api/v1/tts', {method: 'POST', body: params})
         .then((response) => response.text())
         .then((ttsLink) => document.getElementById('audio')
             .setAttribute('src', ttsLink));
   });
 
   return (
-    <Card className={classes.root}>
-      <CardHeader
-        title="Artwork Name"
-        subheader="Artist Name"
-        subheaderTypographyProps={subheaderTypographyProps}
-        className={classes.header}
-      />
-      <Grid
-        container
-        direction="column"
-        justify="center"
-        alignItems="flex-start"
-        spacing={4}
-        className={classes.root}
-      >
-        <Grid item xs={12} md={8}>
-          <CardMedia
-            className={classes.media}
-            image={PlaceholderImage}
-            title="Artwork"
-          />
-          <CardContent className={classes.content}>
-            <Typography
-              variant="body2"
-              color="textSecondary"
-              align="center"
-              component="p"
-            >
-              This a is short physical description of the artwork.
-            </Typography>
-            <AudioPlayer controls id='audio' className={classes.audioPlayer}/>
-          </CardContent>
+    <Container className={classes.withPadding}>
```

Comment:
The difference here is that I wrapped the component in a Container here instead of Dashboard so that I could pass ArtworkCloseUpCard as a prop for Route